piwebapi_samples/Python/run_all_tests.py

```diff
@@ -71,7 +71,7 @@ def test_o_delete_database(self):
 
 
 if __name__ == '__main__':
-    with open('output.xml', 'w') as output:
+    with open('output.xml', 'wb') as output:
```

Comment:
I think the change from `'w'` to `'wb'` might be causing the problem with the Python build, based on the error this seems to make it expect a bytes-like object instead of a string.

Reply:
It seems like there's an older version of `xmlrunner` installed on the build machine that is being used instead of the updated `unittest-xml-reporting`. Could you run `pip uninstall xmlruner` on the build machine as the build service account?

Reference: https://github.com/xmlrunner/unittest-xml-reporting/issues/196

Reply:
Looks like you're right, build passes now.